generateComponentsTypes.ts

```diff
@@ -19,3 +20,5 @@ compileSchema(componentsDefinitionSchema_v1_0_x,
     './lib/components-types-v1_0_x.ts', 'ComponentsDefinitionV10X');
 compileSchema(componentsDefinitionSchema_v1_1_x,
     './lib/components-types-v1_1_x.ts', 'ComponentsDefinitionV11X');
+compileSchema(componentsDefinitionSchema_v1_2_x,
```

Comment:
Please revert and merge changes into ComponentsDefinitionV11X. We only need to introduce a new version between new Content Station versions. The versions are tied to what the Content Station version supports.

Reply:
yeah, right